src/migrations/20190428161016-create-comment.js

```diff
@@ -11,24 +11,24 @@ export default {
       allowNull: false,
       references: {
         model: 'Articles',
-        key: 'id',
+        key: 'id'
       },
       onUpdate: 'CASCADE',
-      onDelete: 'CASCADE',
+      onDelete: 'CASCADE'
     },
     userId: {
       type: Sequelize.INTEGER,
       allowNull: true,
       references: {
         model: 'Users',
-        key: 'id',
+        key: 'id'
       },
       onUpdate: 'CASCADE',
-      onDelete: 'SET NULL',
+      onDelete: 'SET NULL'
     },
     body: {
       type: Sequelize.TEXT,
-      allowNull: false,
+      allowNull: false
```

Comment:
I can see you allowed the body of comment to be null, is it good to allow someone to create a comment with null body?